### ISSUE TITLE
InvalidArgumentException is thrown with ->fixPermissionsWithAcl()

### ISSUE THREAD

**Issue by helios-ag** [EasyCorp/easy-deploy-bundle#25]
Getting exception with acl: 
```sh
 [InvalidArgumentException]            
  The "setfacl" option is not defined.
```
https://github.com/EasyCorp/easy-deploy-bundle/blob/master/src/Deployer/DefaultDeployer.php#L46

I suppose that `$this->getConfig('setfacl'`), should be replaced with `'setfacl'`

**Comment by onokje** [comment]
got the same error

**Comment by fon60** [comment]
There are two PR's with a fix. #28 and #45

**Comment by helios-ag** [comment]
Closing as resolved in #28
